.gitignore

```diff
@@ -10,11 +10,12 @@ files
 **/e2e/videos
 node_modules
 sbin
+/.42c
 # Ignore only top-level package-lock.json
 /package-lock.json
 
 # Ignore Helm subcharts
-charts/**/charts
+charts/**/chartsß
```

Comment:
Not sure how an Erzett ß got tacked onto this line but it shouldn't be there and should be removed.